### ISSUE TITLE
Functionality blocked on some views where scroll/scale is not available

### ISSUE THREAD

**Issue by ekelen** [berty/berty#1984]
This is a true bug wherever the height of this view overflows device height, e.g. in landscape mode and/or on very on small devices.

- [x] QR code
- [ ] Main modal (Blocked by #1933 )

1. No QR code, no share button ☹️
<img src="https://user-images.githubusercontent.com/24300177/83053424-fb2bc900-a050-11ea-9752-bc8abd441c6d.png" height=300/>

2. Can't access Add Contact header with add/share buttons; can't close modal by swiping as we can't access the top edge of it
<img src="https://user-images.githubusercontent.com/24300177/83053984-ef8cd200-a051-11ea-908c-d6152e69baad.png" height=300/>

There may be some other cases I haven't found yet.

Blocked by #1933

**Comment by ekelen** [comment]
Do we intend to have full landscape support for every view @zooma2k18 ? If not, we can lock the ListModal into portrait and close this issue.

**Comment by zooma2k18** [comment]
full landscape support is not priority, if you feel it as a quick fix (less than a half day), go for it, if not, to postpone for later.

**Comment by ekelen** [comment]
Closing; I've locked the app in portrait as there are too many issues maintaining basic functionality in landscape. (Agree @zxxma - as discussed, probably better not to support landscape at all for the moment; we don't want to subject anyone to a screen that can't be interacted with.)
